### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by operatino** [MicrosoftDX/Vorlonjs#39]
Get this error on running any `vorlon` command:

```
robert $ vorlon -h
env: node\r: No such file or directory
```

My env: OSx 10.10.2, Node.js version - v0.10.33.

**Comment by palnes** [comment]
Beat me to it by two minutes.
λ ~/src/ vorlon
env: node\r: No such file or directory

This is only when using the npm and is due to Windows line-endings. Changing the line endings from Windows (CRLF / \r\n) to Unix (LF / \n) will fix this issue. Checking out the project from git provides the proper line endings, and will not display this error.

**Comment by operatino** [comment]
Yeah, I already figured out this as well. Needs to be fixed in the package itself.

**Comment by meulta** [comment]
Yep we will fix that, thanks :-)
